install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+set -eo pipefail
+IFS=$'\n\t'
+
+function assert_file_exists() {
+    if ! [ -f "$1" ]; then
+        echo -e "\e[31mCould not find file \"$1\", halting...\e[0m"
+        exit 1
+    fi
+}
+
+assert_file_exists "way-cooler"
+assert_file_exists "way-cooler-bg"
+
+if ! [[ $(id -u) = 0 ]] && ! [[ $# == 1 ]]; then
+    echo -e "\e[31m"
+    echo "The install script should be ran as root!"
```

Comment:
The confusing system of English strikes again! (or rather, my poor memorization of grammar rules)